apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -90,6 +89,48 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             isRequired: true,
         },
 
+        formula: {
+            schemaDoc: 'Calculation formula. Example: balance + charge + recalculation + privilege + penalty',
+            type: Text,
+            access: access.readOnlyAccess,
+        },
+
+        charge: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of money that charged by paid period',
+            access: access.readOnlyAccess,
+        },
+
+        balance: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Recipient balance on the receipt creation moment',
+            access: access.readOnlyAccess,
+        },
+
+        recalculation: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Recipient balance recalculation in case of overpaid or etc',
+            access: access.readOnlyAccess,
+        },
+
+        privilege: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Special privileges for recipient',
+            access: access.readOnlyAccess,
+        },
+
+        penalty: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of money that recipient doesn\'t pay for previous receipt',
+            access: access.readOnlyAccess,
+        },
+
+        paid: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of money that recipient already paid by current receipt',
+            access: access.readOnlyAccess,
+        },
+
         toPayDetails: TO_PAY_DETAILS_FIELD,
```

Comment:
Please 

1. Mark this field as deprecated
2. Create TODO and add JIRA TASKS to migrate these fields to toPayDetails
3. Write a support note to partners and other developers